app/pb_kits/playbook/pb_textarea/_textarea_mixin.scss

```diff
@@ -0,0 +1,46 @@
+@import "../tokens/colors";
+@import "../tokens/border_radius";
+@import "../tokens/spacing";
+
+@mixin pb_textarea($border_color, $background_color, $text_color){
+    @include pb_title_4;
+    border: 1px solid $border_color;
+    border-radius: $border_rad_light;
+    display: block;
+    width: 100%;
+    color: $text_color;
+    background: $background_color;
+    padding: $space_xs $space_sm;
+    transition: 0.2s all;
```

Comment:
@markiearnold should this use one of the animation variables?

Reply:
I changed transition to $easeIn for now, as transitions are going to be further explored.